### PR TITLE
Add PNRR instructions, see #1249

### PR DESCRIPTION
**Pull request by mfortini**
Tentative implementation

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Feat: Add PNRR instructions, see #1249
- subject must not be sentence-case, start-case, pascal-case, upper-case
- type must be lower-case
- type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c83e66f99dca3e08d3e67124847aefe90be1c020
</p>

**Comment by bfabio** [comment]
I think we should also add a snippet with an example

**Comment by bfabio** [comment]
I think we should also add a snippet with an example

File: it/riuso/pubblicazione.md [hunk 1]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
 1. compilare ed includere il file `publiccode.yml` nel repositorio (lasciando vuoto il campo `codiceIPA`);
 2. aggiungere l'URL del repositorio nella [lista](https://github.com/italia/developers-italia-backend/blob/master/crawler/whitelist/thirdparty.yml) aprendo una pull request su GitHub.
 
+## PNRR: soluzioni a Catalogo
+
+Nell'ottica di contribuire al raggiungimento degli obiettivi del Piano Nazionale di Ripresa e Resilienza (PNRR), 
+    e in particolare quelli previsti dalla Missione 1, in capo al Dipartimento per la trasformazione digitale, 
+    abbiamo implementato alcuni percorsi di ricerca tematici che agevolino le Pubbliche Amministrazioni a 
+    reperire soluzioni rispondenti ai requisiti indicati negli avvisi pubblicati da [PA Digitale 2026](https://padigitale2026.gov.it/).
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:59]:
```suggestion
Nell'ottica di contribuire al raggiungimento degli obiettivi del Piano Nazionale di Ripresa e Resilienza (PNRR), 
e in particolare quelli previsti dalla Missione 1, in capo al Dipartimento per la trasformazione digitale, 
abbiamo implementato alcuni percorsi di ricerca tematici che agevolino le Pubbliche Amministrazioni a 
reperire soluzioni rispondenti ai requisiti indicati negli avvisi pubblicati da [PA Digitale 2026](https://padigitale2026.gov.it/).

```

File: it/riuso/pubblicazione.md [hunk 2]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+Nella ricerca di soluzioni che possano concorrere al raggiungimento di tali requisiti, va rispettato quanto previsto dal 
+Codice dell'Amministrazione Digitale (Artt. 68 e 69) effettuano una _valutazione comparativa_ che dia priorità,
+nell'ordine, al software a riuso ed open source di terze parti.
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:62]:
```suggestion
nell'ordine, al software a riuso ed open source di terze parti.

```

File: it/riuso/pubblicazione.md [hunk 3]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+
+Nell'ottica di contribuire al raggiungimento degli obiettivi del Piano Nazionale di Ripresa e Resilienza (PNRR), 
+    e in particolare quelli previsti dalla Missione 1, in capo al Dipartimento per la trasformazione digitale, 
+    abbiamo implementato alcuni percorsi di ricerca tematici che agevolino le Pubbliche Amministrazioni a 
+    reperire soluzioni rispondenti ai requisiti indicati negli avvisi pubblicati da [PA Digitale 2026](https://padigitale2026.gov.it/).
+Nella ricerca di soluzioni che possano concorrere al raggiungimento di tali requisiti, va rispettato quanto previsto dal 
+Codice dell'Amministrazione Digitale (Artt. 68 e 69) effettuano una _valutazione comparativa_ che dia priorità,
+nell'ordine, al software a riuso ed open source di terze parti. 
+Per agevolare la ricerca di soluzioni idonee abbiamo individuato alcune informazioni che possono essere inserite, su base volontaria e autocertificata, nel file `publiccode.yml`, utilizzato
+per la pubblicazione delle schede di presentazione del Catalogo. Per autocertificare la propria soluzione è necessario seguire le istruzioni di seguito riportate.
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:64]:
```suggestion
Per agevolare la ricerca di soluzioni idonee abbiamo individuato alcune informazioni che possono essere inserite, su base volontaria e autocertificata, nel file `publiccode.yml`, utilizzato
per la pubblicazione delle schede di presentazione del Catalogo.
```

File: it/riuso/pubblicazione.md [hunk 4]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+
+### Istruzioni
+
+All'interno del file `publiccode.yml`, nel campo `features` si chiede di inserire i tag:
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:68]:
```suggestion
All'interno del file `publiccode.yml`, nel campo `features` inserire:
```

File: it/riuso/pubblicazione.md [hunk 5]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+
+* `PNRR` per indicare che il software appartiene alla categoria
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:70]:
```suggestion
* "`PNRR`" per indicare che il software appartiene alla categoria
```

File: it/riuso/pubblicazione.md [hunk 6]
```diff
@@ -51,3 +51,37 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+* `PNRR/Beneficiari/{NOME BENEFICIARIO}` per indicare che il software può avere come beneficiari quelli indicati. (Questo valore può essere ripetuto tante volte quanto necessario). Esempio: `PNRR/Beneficiari/Scuole`
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:71]:
```suggestion
* "`PNRR/Beneficiari/{NOME BENEFICIARIO}`" per indicare che il software può avere come beneficiari quelli indicati. Questo valore può essere ripetuto tante volte quanto necessario.

  Esempio: "`PNRR/Beneficiari/Scuole`"
```

File: it/riuso/pubblicazione.md [hunk 7]
```diff
@@ -51,3 +51,41 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+
+### Istruzioni
+
+All'interno del file `publiccode.yml`, nel campo `features` si chiede di inserire i tag:
+
+* "`PNRR`" per indicare che il software appartiene alla categoria
+* "`PNRR/Beneficiari/{NOME BENEFICIARIO}`" per indicare che il software può avere come beneficiari quelli indicati. Questo valore può essere ripetuto tante volte quanto necessario.
+
+  Esempio: "`PNRR/Beneficiari/Scuole`"
+* `PNRR/Misura/{NUMERO MISURA}` per indicare che il software può essere utile per rispondere alla misura numero `{NUMERO MISURA}`. (Questo valore può essere ripetuto tante volte quanto necessario). Esempio: `PNRR/Misura/1.2`
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:76]:
```suggestion
* `PNRR/Misura/{NUMERO MISURA}` per indicare che il software può essere utile per rispondere alla misura numero `{NUMERO MISURA}`. (Questo valore può essere ripetuto tante volte quanto necessario).
  
  Esempio: "`PNRR/Misura/1.2`"
```

File: it/riuso/pubblicazione.md [hunk 8]
```diff
@@ -51,3 +51,43 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+per la pubblicazione delle schede di presentazione del Catalogo. Per autocertificare la propria soluzione è necessario seguire le istruzioni di seguito riportate.
+
+### Istruzioni
+
+All'interno del file `publiccode.yml`, nel campo `features` si chiede di inserire i tag:
+
+* "`PNRR`" per indicare che il software appartiene alla categoria
+* "`PNRR/Beneficiari/{NOME BENEFICIARIO}`" per indicare che il software può avere come beneficiari quelli indicati. Questo valore può essere ripetuto tante volte quanto necessario.
+
+  Esempio: "`PNRR/Beneficiari/Scuole`"
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:75]:
```suggestion
  Esempio: "`PNRR/Beneficiari/Scuole`"
  
  I possibili nomi di beneficiari sono:
    - "`Comuni`"
    - "`Scuole`"
    - "`ASL`"
    - "`Enti Regionali`"
    - "`Università`"
    - "`PA Centrali`"
    - "`Province`"
    - "`Istituti di ricerca e AFAM`"
    - "`Altri enti`"
```

File: it/riuso/pubblicazione.md [hunk 9]
```diff
@@ -51,3 +51,54 @@ I software open source di terze parti, ovvero non messi a riuso dalla Pubblica A
...
+
+    description:
+      it:
+        features:
+          - PNRR
+          - PNRR/Misura/1.4.1
+          - PNRR/Beneficiari/Comuni
+
+
+Il Catalogo presenta quindi [una nuova voce di ricerca "PNRR"](/it/search?pnrr=1), che permette di filtrare le soluzioni per ciascuno dei tre elementi sopra riportati.
```
Comment (bfabio) [inline it/riuso/pubblicazione.md:102]:
```suggestion
Il Catalogo presenta [un filtro di ricerca "PNRR"](/it/search?pnrr=1), che permette di trovare le soluzioni per ciascuno dei tre elementi sopra riportati.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**